server/services/v1/api.go

```diff
@@ -40,10 +39,10 @@ import (
 )
 
 const (
-	databasePath        = "/databases"
-	databasePathPattern = databasePath + "/*"
+	projectPath        = "/projects"
+	projectPathPattern = projectPath + "/*"
 
-	collectionPath        = databasePath + "/collections"
+	collectionPath        = projectPath + "/collections"
```

Comment:
`collectionPath        = projectPath + "database/collections"`

Reply:
that will evaluate to 

`/v1/projects/database/collections` 

I think we can just listen at `/v1/projects/*` and  `/v1/info` for api.go  - wdyt?

Reply:
but for collection APIs, we do want to listen to the right path, no?

Reply:
synced offline and switched to

```
	collectionPath        = projectPathPattern + "/database/collections"

```